gapic-generator-cloud/templates/cloud/client/method/def/_normal.erb

```diff
@@ -2,16 +2,32 @@
 ##
 <%- if method.doc_description -%>
 <%= indent method.doc_description, "# " %>
+#
+<%- end -%>
+# @overload <%= method.name %>(request, options: nil)
+#   @param request [<%= method.request_type %> | Hash]
+<%- if method.doc_description -%>
+<%= indent method.doc_description, "#     " %>
 <%- end -%>
+#   @param options [Google::Gax::CallOptions]
+#     Overrides the default settings for this call, e.g, timeout, retries, etc.
 #
-<%- method.fields.each do |field| -%>
-# @param <%= field.name %> [<%= field.doc_types %>]
-<%- if field.doc_description -%>
-<%= indent field.doc_description, "#   " %>
+<%- arg_list = method.arguments.map { |arg| "#{arg.name}: nil"}.join ", " -%>
+# @overload client_method(<%= arg_list %>, options: nil)
```

Comment:
Similarly client_method should be wrapped in erb tags.

Reply:
Fixed.